dvc/remote/base.py

```diff
@@ -39,6 +39,9 @@
 }
 
 
+CHECKSUM_JOBS = min(1, max(4, cpu_count() / 2))
```

Comment:
This was suggested by @efiop in #2073:

> 4 by default seems like a little too much in general case. Maybe we could use NCPU and decide based on that so that (except 1CPU case) we don't take more than 50% of CPUs by default?